src/components/Dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function Dashboard() {
+
+    return (
+      <div><h1 class="text-center">Dashboard Component</h1></div>
```

Comment:
lets change this to className

Reply:
Ramon. Great catch! I accidentally used the normal syntax instead of React syntax. I’ll take care of shortly. Thank you.